pysc2/env/sc2_env.py

```diff
@@ -259,7 +261,7 @@ def __init__(self,  # pylint: disable=invalid-name
     self._run_config = run_configs.get()
     self._parallel = run_parallel.RunParallel()  # Needed for multiplayer.
 
-    interface = sc_pb.InterfaceOptions(raw=visualize, score=True)
+    interface = sc_pb.InterfaceOptions(raw=(visualize or feature_units), score=True)
```

Comment:
Follow the style guide: https://google.github.io/styleguide/pyguide.html . This is over 80 chars long.